utils/cp_parallel.py

```diff
@@ -98,6 +109,11 @@ def run_cellprofiler_parallel(
             "-i",
             path_to_images,
         ]
+
+        # if plugins_dir is provided, add the flag to find the plugins directory with given path
+        if plugins_dir:
+            command.extend(["--plugins-directory", plugins_dir])
```

Comment:
```suggestion
        # if plugins_dir is provided, add the flag to find the plugins directory with given path
        if plugins_dir:
            if not pathlib.Path(plugins_dir).is_dir():
                raise FileNotFoundError(
                    f"Directory '{pathlib.Path(plugins_dir).name}' does not exist or is not a directory"
                    )
            else:
                command.extend(["--plugins-directory", plugins_dir])
```

Note that I have not tried this yet

Reply:
I can easily add this! I do think that the resolve function also works but if someone forgets then this will catch it. I will add this outside of the for loop in the function to avoid any issues.

Reply:
Just tested this and it works! 😄